charts/k8s-ovpn-chart/.helmignore

```diff
@@ -20,4 +20,3 @@
 .idea/
 *.tmproj
 .vscode/
-ovpn0
```

Comment:
I would keep ignoring this directory just to make sure it's not getting accidentally packed in a chart